tests/shared.py

```diff
@@ -69,7 +69,11 @@ def __call__(self, *args, **kwargs):
 
 @MemoizedSupplier
 def get_driver_features(backend):
-
+    # TODO: remove this as soon as java supports GetFeatures
+    if get_driver_name() in ["java"]:
```

Comment:
See https://github.com/neo4j/neo4j-java-driver/pull/889#issuecomment-838462619

Reply:
But it's not merged yet. So all pipelines until then will be blocked :/